engine/clients/qdrant/search.py

```diff
@@ -34,11 +35,23 @@ def init_client(cls, host, distance, connection_params: dict, search_params: dic
     #     return "forkserver" if "forkserver" in mp.get_all_start_methods() else "spawn"
 
     @classmethod
-    def search_one(cls, vector, meta_conditions, top) -> List[Tuple[int, float]]:
+    def search_one(cls, query: Query, top) -> List[Tuple[int, float]]:
+        # Can query only one till we introduce re-ranking in the benchmarks
+        if query.sparse_vector is None:
+            query_vector = query.vector
+        else:
+            query_vector = rest.NamedSparseVector(
```

Comment:
Minimum: I would use .construct with all vector structures

Better: build grpc structures straightaway to avoid conversions, since we are primarily testing client with grpc, rather than with rest

Reply:
Done